pkgs/applications/misc/actiona/default.nix

```diff
@@ -0,0 +1,104 @@
+{
+  lib,
```

Comment:
Is this comma-after-args a new obligatory style?

Reply:
See https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax

Reply:
Please take a look at https://github.com/NixOS/rfcs/pull/166 if you haven't already.
Esentially, there will be a standardized nix formatter, which is still being worked on, but the implementation is almost complete: it's available as `nixfmt-rfc-style`. I am using it as the formatter for my newly created PRs now.
The entire repo will be mass-formatted with it in the future, so I'm using it to avoid having to do cleanup PRs.

Reply:
OK

Reply:
> nixfmt-rfc-style

Oh, didn't know.